enhancements/update/clusterversion-osus-signatures.md

```diff
@@ -0,0 +1,138 @@
+---
+title: ClusterVersion-has-option-to-reach-local-OSUS-instance-for-signature-verification
+authors:
+  - "@PratikMahajan"
+reviewers:
+  - "@LalatenduMohanty"
+  - "@wking"
+approvers:
+  - "@sdodson"
```

Comment:
@PratikMahajan You can change it to my name. As this only impacts CVO , I can approve it after we get API team's review comment.